pkg/ui/workspaces/db-console/src/views/cluster/containers/nodeGraphs/dashboards/storage.tsx

```diff
@@ -230,6 +230,50 @@ export default function (props: GraphDashboardProps) {
       </Axis>
     </LineGraph>,
 
+    <LineGraph
+      title="L0 SSTable Count"
+      sources={storeSources}
+      isKvGraph={true}
+      tenantSource={tenantSource}
+      tooltip={`The number of L0 SSTables in use for each store ${tooltipSelection}.`}
+      showMetricsInTooltip={true}
+    >
+      <Axis label="sstables">
+        {nodeIDs.flatMap(nid => (
+          storeIDsForNode(storeIDsByNodeID, nid).map(sid => (
+            <Metric
+              key={`${nid}-${sid}`}
+              name="cr.store.storage.l0-num-files"
+              title={`store ${sid} ${getNodeNameById(nid)}`}
```

Comment:
nit: might be more concise to go with `${getNodeNameById(nid)},s${sid}` so you get `n9,s17` and so on, which would follow the same convention as in the cockroach logs.

Reply:
Makes sense - just updated this.